gobblin-data-management/src/main/java/gobblin/data/management/copy/CopySource.java

```diff
@@ -93,8 +101,11 @@
 
         Path targetRoot = getTargetRoot(state, datasetFinder, copyableDataset);
 
-        Collection<Partition<CopyableFile>> partitions =
-            copyableDataset.partitionFiles(copyableDataset.getCopyableFiles(targetFs, targetRoot));
+        CopyConfiguration copyConfiguration = new CopyConfiguration(targetRoot,
```

Comment:
Is the PRESERVE_ATTRIBUTES_KEY mandatory? Can't we have a sensible default (""), i.e. don't preserve anything?